### ISSUE TITLE
"Generated code, do not edit"

### ISSUE THREAD

**Issue by holiman** [crate-crypto/go-ipa#21]
I noticed this when opening a PR touching https://github.com/crate-crypto/go-ipa/blob/master/bandersnatch/fr/element.go , github won't show the diff by default because it's marked as such: 

https://github.com/crate-crypto/go-ipa/blob/master/bandersnatch/fr/element.go#L15

```
// Code generated by consensys/gnark-crypto DO NOT EDIT
``` 

It seems that some parts are generated and others are not? If so, it would be better to move all non-generated methods out into a separate file, and keep the pure generated code separate in that file. If the header lies, then it should be removed.

**Comment by kevaundray** [comment]
Good point. The only additional methods in this file are BytesLE and SetBytesLE, as you said we can put these into another file.
